src/com/cli/contact/Main.java

```diff
@@ -0,0 +1,129 @@
+package com.cli.contact;
+
+import com.cli.contact.manager.SearchLastName;
+import com.cli.contact.manager.SearchOnlyLetter;
+import com.cli.contact.manager.SearchPhone;
+import com.cli.contact.manager.Searcher;
+import com.cli.contact.model.Contact;
+import com.cli.contact.model.Person;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Person p1 = new Person("angelica","lopez","anghe@gmail.com",4287052);
+        Person p2 = new Person("daniel","cespedes","daniel@gmail.com",2287052);
+        Person p3 = new Person("carlos","mayta","carlos@gmail.com",3287052);
+        Person p4 = new Person("sofia","perez","sofi@gmail.com",5287052);
+
+        Contact contact = new Contact();
+        contact.addContact(p1);
+        contact.addContact(p2);
+        contact.addContact(p3);
+        contact.addContact(p4);
+
+            int choice = 0;
+
+            while (choice != 4)
+            {   System.out.println();
+                showMenu();
+                Scanner scanner = new Scanner(System.in);
+                choice = scanner.nextInt();
+
+                if (choice == 1) {
```

Comment:
@anghelitalo for this case when you have multiple ifs it is better to use the switch case. you can review this https://stackoverflow.com/questions/2103080/when-to-use-a-switch-statement-in-java